builtin/game/auth.lua

```diff
@@ -53,7 +53,31 @@ local function save_auth_file()
 	end
 end
 
+local function append_auth_file(name)
+	local stuff = auth_table[name]
+	-- Check table for validness before attempting to save
+	assert(type(name) == "string")
+	assert(name ~= "")
+	assert(type(stuff) == "table")
```

Comment:
`stuff` is nil when called from `delete_auth(...)`

Reply:
Good point – I hadn't tested `delete_auth`…

… which reminds me that I had wanted to handle deletes, but completely forgot! D'oh!